frontend/pages/queries/edit/components/SaveQueryModal/SaveQueryModal.tsx

```diff
@@ -315,7 +238,17 @@ const SaveQueryModal = ({
               label="Logging"
               wrapperClassName={`${baseClass}__form-field ${baseClass}__form-field--logging`}
             />
-            {!isLoadingAppConfig && renderDiscardDataOption()}
+            {!isLoadingAppConfig && (
+              <DiscardDataOption
+                {...{
+                  appConfig,
+                  selectedLoggingType,
+                  discardData,
+                  setDiscardData,
+                }}
+                breakHelpText
```

Comment:
As mentioned above, moving `breakHelpText` into the object above would make it easier for the reader to keep everything grouped together in their head.